engine/time/src/main/java/io/deephaven/time/DateTimeUtils.java

```diff
@@ -1438,9 +1440,7 @@ public static ZonedDateTime excelToZonedDateTime(final double excel, @Nullable f
         if (micros == QueryConstants.NULL_LONG) {
             return null;
         }
-        final long microsPerSecond = 1_000_000;
-        return LocalDateTime.ofEpochSecond(micros / microsPerSecond, (int) ((micros % microsPerSecond) * MICRO),
-                ZoneOffset.UTC);
+        return epochNanosToLocalDateTimeUTC(micros * MICRO);
```

Comment:
I was wrong; the previous code better matches our existing implementations. Below matches "epochMicrosToInstant`:
```
        return micros == NULL_LONG ? null : LocalDateTime.ofEpochSecond(micros / 1_000_000L, (int) ((micros % 1_000_000L) * MICRO), ZoneOffset.UTC);
```

And the following for millis:
```
return millis == NULL_LONG ? null : LocalDateTime.ofEpochSecond(millis / 1_000L, (int) ((millis % 1_000L) * MILLI), ZoneOffset.UTC);
```

Reply:
This would argue for the nanos one to be:
```
        return nanos == NULL_LONG ? null : LocalDateTime.ofEpochSecond(nanos / 1_000_000_000L, (int) (nanos % 1_000_000_000L), ZoneOffset.UTC);
```